src/components/Expandable/ExpandableSlots.svelte

```diff
@@ -0,0 +1,63 @@
+<script>
+  import { onMount, onDestroy } from 'svelte';
+  import Portal from '../Portal.svelte';
+  import { expandAnimation } from './helpers';
+
+  let parentRef = null;
+  let isExpanded = false;
+  let isFullVisible = false;
+
+  function showFullScreenComponent() {
+    console.log('-->', 1);
```

Comment:
oh, nice! :) I like such information on console in production code! ))

Reply:
Yeah, It's really important stuff!